### DISCUSSION TITLE
Scraping selected/multiple models?

### DISCUSSION THREAD

**Discussion by kavinklain** [UltimaHoarder/UltimaScraper#1631] [Q&A]
so as of right now entering 0 will scrape all models you've subscribed to, but is there a way to select and scrape multiple models? ie  i just subbed to 2-3 new models due to a sale and I want to just scrape all 3 at once, could I do that or would I have do one at a time?

**Comment by americanseeder1865** [comment]
I would do it all at the same time. Doesn't hurt anything unless you have space constraints.

**Reply by kavinklain** [reply]
when you say all at the same time do you mean selecting all ie 0, when given the model choice? I noticed that this still makes you go through each model and to choose what you want scraped
